docs/releases/ibexa_dxp_v4.0.md

```diff
@@ -82,20 +82,20 @@ See [Managing migrations](../api/public_php_api_managing_migrations.md) for more
 Alternative text for an Image field is now optional by default. 
 You can set it as required when adding the Image field to a Content Type.
 
-### Configure what elements are available in the Page Builder for the Content type
+### Configure what elements are available in the Page Builder for the Content type [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
 
 You can now select which page blocks, page layout and what edit mode are available in the Editor mode for the Content type.
 For more information, see [Working with Page](https://doc.ibexa.co/projects/userguide/en/latest/site_organization/working_with_page/#configure-blocks-display).
 
-### Purge all submissions of given form
+### Purge all submissions of given form [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
 
 You can purge all submissions of a given form. 
 For more information, see [Forms](../guide/form_builder/forms.md#form-submission-purging).
 
 ### Hidden eCommerce features
 
-Commerce tab and all its features are now disabled by default.
-For more information, see [Enable Commerce features](../guide/config_back_office.md#enable-commerce-features).
+In [[= product_name_content =]], Commerce tab and all its features are now disabled by default.
+For more information, see [Enabling Commerce features](https://doc.ibexa.co/en/4.0/guide/config_back_office/#enable-commerce-features).
```

Comment:
Watch out when merging to earlier branches: you're removing the section linked here in this PR.

Reply:
https://github.com/ezsystems/developer-documentation/pull/1637/files#diff-ee2c7e1cf422bdfac9c8a1035acd8fdbeecebab1dbb8aa03a292d49f2041d56bL209 removes this section, so after this is merged do 4.0, the link above will not work